molgenis-metadata-manager/src/main/frontend/src/store/actions.js

```diff
@@ -32,7 +32,12 @@ export default {
     get({apiUrl: '/api'}, '/v2/sys_md_EntityType?num=10000')
       .then(response => {
         const nonSystemEntities = response.items.filter(function (item) {
-          return !item.package.id.startsWith('sys')
+          // TODO we should filter on parent package is sys, a user can also select a package name starting with sys_
+          if (item.package) {
+            return !item.package.id.startsWith('sys_')
```

Comment:
Now we can still edit metadata for sys_StaticContent..

But I agree, blocking "sys" will also remove user entities in package system_analysis
This is the real TODO / FIXME